.gitignore

```diff
@@ -25,6 +25,7 @@ catalogd.yaml
 
 # Kubernetes Generated files - skip generated files, except for vendored files
 
+vendor/
 !vendor/**/zz_generated.*
```

Comment:
Y'all reckon we need to keep this?  Won't it be generated anyway?

Reply:
We can test it out to see if I do not know this one.
I will defer this to others

Reply:
I don't think we need to keep `!vendor/**/zz_generated.*`. Looks like something that was cargo-culled from somewhere else.

Reply:
I do not see why we would as well.
So, if we have not an historical context here

/lgtm
/approve